test/formatters.test.ts

```diff
@@ -0,0 +1,34 @@
+import { it, describe, expect } from 'vitest';
+import { addressToNoirFormat, numberToNoirBytes32 } from '../formatters';
+
+describe('addressToNoirFormat()', () => {
+  it("should convert address to noir format", () => {
+    const converted = addressToNoirFormat('0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045');
+    expect(converted).toStrictEqual([
```

Comment:
`converted` seems like a weird name here.
I propose `addressInNoirFormat` or `actual` or even better - inline

Reply:
Make sense, made it inline